app/models.py

```
@@ -747,6 +754,85 @@ class InstanceMetadata(Base):
+    DISCONNECTED = "DISCONNECTED"
+
+
+class P2PNetworkConfig(Base):
+    """Single-node P2P network configuration."""
+
+    __tablename__ = "p2p_network_config"
+
+    id = Column(Integer, primary_key=True, index=True)
+    network_name = Column(String, nullable=False, default="File Fridge P2P")
+    psk_hash = Column(String, nullable=False, unique=True, index=True)
+    listen_host = Column(String, nullable=False, default="0.0.0.0")
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)